kubernetes/cray-uan-install/Chart.yaml

```diff
@@ -21,16 +21,28 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
-apiVersion: v1
-description: "Kubernetes resources for cray-uan-install"
-name: "cray-uan-install"
-home: "scms/uan"
-version: 1.3.0
+apiVersion: v2
+name: cray-uan-install
+version: 1.4.0
+description: Kubernetes resources for installing UAN
 keywords:
   - uan
   - cfs
-sources:
-  - "https://stash.us.cray.com/projects/SCMS/repos/uan/browse"
+home: https://github.com/Cray-HPE/uan
+dependencies:
+  - name: cray-import-config
+    version: 2.0.0
+    repository: https://artifactory.algol60.net/artifactory/csm-helm-charts/
+  - name: cray-import-kiwi-recipe-image
```

Comment:
UAN doesn't ship an image recipe or image any longer.  At least not until we come up with a non-COS based image.  We simply use the COS installed image as a base.  We shouldn't need this dependency.